app/services/images/optimizer.rb

```diff
@@ -81,5 +93,11 @@ def self.get_imgproxy_endpoint
         ApplicationConfig["IMGPROXY_ENDPOINT"] || "http://localhost:8080"
       end
     end
+
+    def self.extract_suffix_url(full_url)
+      uri = URI.parse(full_url)
+      match = uri.path.match(%r{https?://.+})
+      match[0] if match
```

Comment:
I think we need a fallback here if the extract fails. What if we return the `full_url` instead?

Reply:
@maestromac I think nil is fine here. I've added a few tests to handle alternate situations and the worst case is that we ultimately output the prefix only, which is okay because the input was off.

I don't think there's a situation where we fail here.